tools/lib/gcp-token.sh

```diff
@@ -1,69 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Are you sure this isn't being used by any processes (even manual ones)?

Reply:
it is not used any another script. This lib was developed  by me before. and the `ci_credentials.sh` script used it only. (I've checked)